ref/gradescope.md

```diff
@@ -23,45 +23,54 @@ Open your browser and go to <https://www.gradescope.com>
+OR choose "log in in with school credentials" and use your ucsb netid to log in.
 
 ![login using netid](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s1_login_netid.png)
 
 ### Step 2. Choose the course
 
-After logging in, you should able to see one course or a list of courses. Choose the CS8 course for Spring 23.
+After logging in, you should able to see one course or a list of courses. Choose the **CSW8** course.
 
 ![courses](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s2_course.png)
 
 ### Step 3. Choose the assignment to submit
 
-You should see Gradescope labs that you were assigned there. You will click on the one submit your work for that particular lab.
+You should see Gradescope labs that you were assigned there. You may click on one of them and submit your work for that particular lab.
 
 ![assignments](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s3_assignments.png)
 
 
-### Step 4. Submit: upload your code
+### Step 4. Submit your assignment
 
-When click and open one of the labs, if you haven't submitted anything for an assignment yet, Gradescope will prompt you (with a box and submit button) to upload your files. You either can either click inside the box and navigate to your file(s) or "drag-and-drop" them into the "Submit Programming Assignment" window.
+When click and open one of the labs, if you haven't submitted anything for an assignment yet, Gradescope will prompt you (with a box and submit button) to upload your files. You can either click inside the box and navigate to your file(s) or "drag-and-drop" them into the "Submit Programming Assignment" window.
 
 ![submit](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s4_submit.png)
 
+### Step 5. Upload multiple files at once
 
-### Step 5. Resubmit (if needed)
+To select multiple files together:
+* Mac: hold down `command` key while clicking on the files you would like to add
+* Windows: hold down `ctrl` key while clicking on the files you would like to add
+
+![multiple_upload](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s5_multiple_upload.png)
+
+
+### Step 6. Resubmit (if needed)
 
 If you already submitted something on Gradescope, it will take you to their "Autograder Results" page. For most labs, if everything is correct, you'll see a successful submission passing all of the autograder tests.
 
+![successful_submit](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s6_sucessful_submit.png)
 
 There is a **"Resubmit" button** on the bottom right that will allow you to update the files for your submission. You have an unlimited number of submissions up until the lab deadline - we will use the grade of the LAST submitted solution you put in! We recommend submitting your assignment early to make sure that you pass all tests.
 
 
-![resubmit](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s5_resubmit.png)
+![resubmit](https://ucsb-csw8.github.io/s23/assets/images/gradescope/s6_resubmit.png)
```

Comment:
It would be helpful to draw an arrow pointing at the **Resubmit** button. Light green seems easy to miss.

@kimiaf1998, I updated your role to be a student in the Sandbox site, so that you can screenshot the interface the way it actually looks to students.